### ISSUE TITLE
Future of Identity Hub and where to look for updates and information next

### ISSUE THREAD

**Issue by xtianus79** [decentralized-identity/secure-data-store#2]
@csuwildcat I am looking for this becoming a reality for many many reasons. In particular I have a solution and idea that would rely on this type of personal Data store.  

I have a few questions 

1. Where is Identity Hub going to move to once this is archived
2. What is Identity Hub officially being turned into? 
3. How will it transition and any information regarding that transition will be helpful. 
4.  *** Lastly AND most importantly when will a public SDK be available for this new technology? 

I want to build something to this vision even if it is not officially released in the interim.  If I can assure myself an SDK will be available within 12 months I would be able to make my decision on pursing a project for this.

**Comment by csuwildcat** [comment]
Let me try and answer from the POV of what I hope we are able to do with SDS and how that relates to Identity Hubs:

In general, my goal with the SDS WG, and the spec it outputs, is that it codifies the base layers of what a Hub is/was to ensure it has the right foundation/features to support the higher level interfaces that made up Hubs. Once that spec is in place, Hubs should really just be a set of interfaces and logical code mechanisms built on top that produce the kinds of APIs/functionality we all desired in Hubs to begin with.

As for your questions:

1. I think @OR13 probably has a clearer answer for this, but I believe the Hubs stuff is being merged into this new repo as a sort of historical appendix.
2. Identity Hubs will still exist at a future point, but that won't happen until we have the SDS spec done and can build something like the Hub atop its standard foundation.
3. We will try to ensure all the materials, decisions, and content about the transition is open and available.
4. I can't say for sure, but we will be pushing hard to have a rough implementation available in less than a year.

**Comment by OR13** [comment]
Good questions, I think the sooner we get the basics setup the sooner we will have better answers.

I agree with everything @csuwildcat said.

There will be an sdk and an implementation in this repo, and there will be tests, but its hard to say when that will all happen.

**Comment by OR13** [comment]
@xtianus79 you are responsible for moving this issue forward, please close it if you feel it is addressed, or feel free to ask questions / open a more specific issue. I recommend closing this issue.

**Comment by OR13** [comment]
Closing due to inactivty.
